### PR TITLE
Update the list of contributors to 3.3.0

### PR DESCRIPTION
**Pull request by MaxGekk**
In the PR, I propose to add the list of contributors to 3.3.0 that were generated by:
```
$ git shortlog v3.2.1..v3.3.0 -s |cut -f 2 > ./contributors.txt
$ ./translate-contributors.py --non-interactive
```

### CODE REVIEW

**Review by srowen — APPROVED** [review]

**Review by dongjoon-hyun — APPROVED** [review]
There was some discussion about Group Account, `dgd-contributor`, before in the mailing thread. I'm wondering how can we handle `dgd-contributor`.

- https://github.com/dgd-contributor

+1, LGTM for the others.

**Review by viirya — APPROVED** [review]

File: releases/_posts/2022-06-16-spark-release-3-3-0.md [hunk 1]
```diff
@@ -464,4 +464,8 @@ Support the GCM mode by `<span style="text-decoration:underline;">aes_encrypt</s
...
...talii Li, Wang, Warren Zhu, Weichen Xu, Weiwei Yang, Wenchen Fan, William Hyun, Wu, Xiaochang, Xianjin YE, Xiduo You, Xingbo Jiang, Xinrong Meng, Xinyi Yu, XiuLi Wei, Yang He, Yang Liu, YangJie, Yannis Sismanis, Ye Zhou, Yesheng Ma, Yihong He, Yikf, Yikun Jiang, Yimin, Yingyi Bu, Yuanjian Li, Yufei Gu, Yuming Wang, Yun Tang, Yuto Akutsu, Zhen Li, Zhenhua Wang, Zimo Li, alexander_holmes, beobest2, bjornjorgensen, chenzhx, copperybean, daugraph, dch nguyen, dchvn, dchvn nguyen, dgd-contributor, dgd_contributor, dohongdayi, erenavsarogullari, fhygh, flynn, gaoyajun02, gengjiaan, herman, hi-zir, huangmaoyang2, huaxingao, hujiahua, jackierwzhang, jackylee-ch, jiaoqb, jinhai, khalidmammadov, kuwii, leesf, mans2singh, mcdull-zhang, michaelzhang-db, minyyy, nyingping, pralabhkumar, qitao liu, remykarem, sandeepvinayak, senthilkumarb, shane knapp, skhandrikagmail, sperlingxx, sudoliyang, sweisdb, sychen, tan.vu, tanel.kiis@gmail.com, tenglei, tianhanhu, tianlzhang, timothy65535, tooptoop4, vadim, w00507315, wangguangxin.cn, wangshengjie3, wayneguow, wooplevip, wuyi, xiepengjie, xuyu, yangjie01, yaohua, yi.wu, yikaifei, yoda-mon, zhangxudong1, zhoubin11, zhouyifan279, zhuqi-lucas, zwangsheng
```
Comment (HyukjinKwon) [inline releases/_posts/2022-06-16-spark-release-3-3-0.md:471]: I usually manually check GitHub page by ID and replace the names though :-).

**Review by HyukjinKwon — COMMENTED** [review]
LGTM2 except one comment

File: releases/_posts/2022-06-16-spark-release-3-3-0.md (hunk 1 above)
Comment (HyukjinKwon) [reply releases/_posts/2022-06-16-spark-release-3-3-0.md:471]: and add to knwon translation (e.g., https://github.com/apache/spark/commit/8d68f3f74658c7e0c12ee1d6f09a1aae14d9e04f)

**Pull request closed without merging**